src/ast/nodes/shared/FunctionBase.ts

```diff
@@ -237,7 +285,10 @@ export default abstract class FunctionBase extends NodeBase {
 		this.included = true;
 		const { brokenFlow } = context;
 		context.brokenFlow = false;
-		this.body.include(context, includeChildrenRecursively);
+		this.body.includePath(UNKNOWN_PATH, context, includeChildrenRecursively);
+		for (const argument of this.argumentsToBeIncludedAll) {
+			argument.includePath(UNKNOWN_PATH, context, false);
+		}
```

Comment:
Previously, including call arguments was solely handled in `includeCallArguments` which would again be performed in the ParameterScope. I wonder if we could still do it there. At the moment, the ParameterScope does not have enough information to do that but I wonder if things could be refactored in some way. I will need to think about this.

Reply:
It seems that we can still do it in `ParameterScope` like in commit https://github.com/rollup/rollup/pull/5420/commits/222722c19b4666ae0f689e375d46f5ecbebc7ab3.